cfpb/cfgov-refresh
# Ask updates

General Ask updates.

## Additions
- Add sibling page link to English and Spanish answer pages.

## Removals
- Remove HTML editor from answer pages.

## Changes
- Update Spanish mega menu and footer links to replace `Inicio` with `Para consumidores`.
- Display extra sidebar content above disclaimer on answer pages.
- Constrain H2 widths on answer pages.

## Testing

1. Check out branch.
2. Run `python cfgov/manage.py migrate` and `gulp styles`
3. Navigate to [an English Ask answer page](http://localhost:8000/ask-cfpb/can-a-lender-or-dealer-ask-me-about-the-alimony-child-support-or-separate-maintenance-payments-that-i-receive-when-i-apply-for-an-auto-loan-en-1195/) and verify the sidebar links between it and its Spanish sibling 
4. Navigate to [a Spanish page](http://localhost:8000/es/) and verify the updated links in the header/footer
5. Open an [answer page admin edit view](http://localhost:8000/admin/pages/7240/edit/) and verify that the html edit link does not appear in its answer content text editors
6. Open an [answer page admin edit view](http://localhost:8000/admin/pages/7240/edit/). Add a long H2 header in a new table block and verify that the header's width is constrained to the width of the surrounding content.
7. Open an [answer page admin edit view](http://localhost:8000/admin/pages/7240/edit/). Add an email block to its sidebar and verify that the email signup appears between the `About us` and `Legal disclaimer` blocks.

## Screenshots

### Spanish answer page with updated menu
![Spanish_answer](https://user-images.githubusercontent.com/778171/58826934-b1073780-85f6-11e9-8d4e-232f41c34f9f.png)


### English answer page with custom sidebar content
![English_page](https://user-images.githubusercontent.com/778171/58826981-cda36f80-85f6-11e9-8a81-49bc7f01e81e.png)
